tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -119,6 +119,17 @@ sub run {
         # but not implemented in file_content_replace() yet poo#120690
         $escaped_token =~ s/\&/\\\&/g;
         set_var("HANA_TOKEN", $escaped_token);
+
+        if (get_var('PTF_INSTALL')) {
+            my $ptf_files = qesap_az_list_container_files(
+                storage => get_required_var('HANA_ACCOUNT'),
+                container => (split("/", get_required_var('HANA_CONTAINER')))[0],
+                token => get_required_var('HANA_TOKEN'),
+                prefix => 'ptf'
+            );
+            set_var("PTF_FILES", $ptf_files);
```

Comment:
is it absolutely needed to do so? To save the list to a setting?

Reply:
In the current implementation it is, because `$ptf_files` is declared and populated inside an `if {}`. Thus, the `$ptf_files` var is not available later, when the function call is using it.

Plus, `get_var` will either return the variable if it exists, or undefined if it doesn't, so it won't lead to failure if it was never set

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19592/files#r1656808897